docs/content/get-started/installation/agent/bare_metal.md

```diff
@@ -45,6 +45,20 @@ curl --fail --location --remote-name "\${url}"
 );
 ```
 
+```mdx-code-block
```

Comment:
This is a wrong place for this as binaries won't be available for bare metal packages of aperture agent but will be available only for aperturectl